### PR TITLE
Build: make JSCS and JSHint lint the grunt config too

### PR DESCRIPTION
**Pull request by arthurvr**
This makes JSHint and JSCS lint the files in `tasks/` too. I asked this but seems like @arschmitz forgot it: https://github.com/jquery/css-chassis/pull/89#discussion_r32540343.

### CODE REVIEW

**Comment by arthurvr** [comment]
The CI failure is on master too.

File: tasks/variables.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 module.exports = function( grunt ) {
 	grunt.registerTask( "buildVariables", function() {
-		var jsass = require( "jsass-vars" );
-		var path = require( "path" );
-		var varPath = path.join( process.cwd(), "scss/variables/*" );
+		var jsass = require( "jsass-vars" ),
```
Comment (arschmitz) [inline tasks/variables.js]: One var is no longer a requirment but we need to update our version of JSCS for it to not complain.

**Comment by arschmitz** [comment]
@arthurvr looks good other then id prefer to update and not use one var any more.

**Comment by arthurvr** [comment]
> not use one var any more.

I considered it unrelated to this PR, but yeah, updated now.

> other then id prefer to update

Can you elaborate?

**Comment by arschmitz** [comment]
@arthurvr I just meant i would prefer to update jscs then leave one var. Thats all i have.

**Comment by robbbz** [comment]
lgtm modulo the ci failure

**Comment by sfrisk** [comment]
Hey @arthurvr, when you have a sec, could you update this with master?

**Comment by arthurvr** [comment]
Yep, I totally can! It'll have to wait ~2 weeks though. It's only a
semicolon.

**Comment by sfrisk** [comment]
Oh if it is just that I will fix it. I didn't look to closely at what was causing it last night.

**Pull request closed without merging**